projects/plugins/jetpack/modules/subscriptions/subscribe-modal/class-jetpack-subscribe-modal.php

```diff
@@ -64,17 +64,35 @@ public function enqueue_assets() {
 			wp_enqueue_script( 'subscribe-modal-js', plugins_url( 'subscribe-modal.js', __FILE__ ), array( 'wp-dom-ready' ), JETPACK__VERSION, true );
 
 			/**
-			 * Filter how many milliseconds a user must scroll for until the Subscribe Modal appears.
+			 * Filter how many milliseconds until the Subscribe Modal appears.
 			 *
 			 * @module subscriptions
 			 *
 			 * @since 13.4
 			 *
 			 * @param int 300 Time in milliseconds for the Subscribe Modal to appear upon scrolling.
```

Comment:
We should update this default comment since it changed.

Reply:
Should we introduce a slightly different filter name since this one might be in use already. 
cc: @Aurorum for awareness and feedback, since you introduced the original filter. :)

Reply:
No strong views from me! It was added to allow users to customise the default value for the load time; if the default value still changes, I don’t think that’s a big deal.

Reply:
> We should update this default comment since it changed.

done!